[BUG] cdk: using `-c` to pass parameters to cdk conflicts with `npx -c <cmd>`

When passing parameters to a CDK module, we are using `-c`.

https://github.com/onicagroup/runway/blob/1b19532acae4ea0ee65f515531f08f19a183dd06/runway/module/cdk.py#L125

This option overlaps with `npx` resulting in the following error:

```
npm ERR! npm exec
npm ERR! 
npm ERR! Run a command from a local or remote npm package
npm ERR! 
npm ERR! Usage:
npm ERR! npm exec -- <pkg>[@<version>] [args...]
npm ERR! npm exec --package=<pkg>[@<version>] -- <cmd> [args...]
npm ERR! npm exec -c '<cmd> [args...]'
npm ERR! npm exec --package=foo -c '<cmd> [args...]'
npm ERR! 
npm ERR! Options:
npm ERR! [--package <pkg>[@<version>] [--package <pkg>[@<version>] ...]]
npm ERR! [-c|--call <call>]
npm ERR! [-w|--workspace <workspace-name> [-w|--workspace <workspace-name> ...]]
npm ERR! [-ws|--workspaces]
npm ERR! 
npm ERR! alias: x
npm ERR! 
npm ERR! Run "npm help exec" for more info
Subprocess exited with error 1
```

We should use the FQN of the option (`--context`) to avoid this conflict.

--- a/runway/module/cdk.py
+++ b/runway/module/cdk.py
@@ -122,7 +122,7 @@ class CloudDevelopmentKit(RunwayModuleNpm):
                         self.logger.info("build steps (complete)")
                     cdk_context_opts: List[str] = []
                     for key, val in cast(Dict[str, str], self.parameters).items():
-                        cdk_context_opts.extend(["-context", "%s=%s" % (key, val)])
+                        cdk_context_opts.extend(["--context", "%s=%s" % (key, val)])
                     cdk_opts.extend(cdk_context_opts)
                     if command == "diff":
                         self.logger.info("plan (in progress)")

--- a/runway/module/cdk.py
+++ b/runway/module/cdk.py
@@ -122,7 +122,7 @@ class CloudDevelopmentKit(RunwayModuleNpm):
                         self.logger.info("build steps (complete)")
                     cdk_context_opts: List[str] = []
                     for key, val in cast(Dict[str, str], self.parameters).items():
-                        cdk_context_opts.extend(["-c", "%s=%s" % (key, val)])
+                        cdk_context_opts.extend(["-context", "%s=%s" % (key, val)])
                     cdk_opts.extend(cdk_context_opts)
                     if command == "diff":
                         self.logger.info("plan (in progress)")

